packages/erebos-api-bzz-base/src/index.js

```diff
@@ -77,4 +77,16 @@ export default class BaseBzz {
   downloadRawText(hash: string): Promise<string> {
     return this.downloadRaw(hash).then(res => res.text())
   }
+
+  listDirectory(hash: string): Promise<*> {
+    return this._fetch(`${this._url}bzz-list:/${hash}`).then(
+      res => (res.ok ? res : Promise.reject(new Error(res.statusText))),
```

Comment:
Rather than returning just `res` could we use `res.json()` to return the parsed body here, assuming it's sending data as JSON?